packages/core-typings/src/IMessage/IMessage.ts

```diff
@@ -155,7 +155,7 @@ export interface IEditedMessage extends IMessage {
 }
 
 export const isEditedMessage = (message: IMessage): message is IEditedMessage =>
-  "editedAt" in message && "editedBy" in message;
+  "editedAt" in message && "editedBy" in message && !!(message as IEditedMessage).editedAt && !!(message as IEditedMessage).editedBy?._id;
```

Comment:
🤔 i can understand the check for editedBy.id, but why do we need the !!msg.editedAt?

Reply:
I was running into a situation where the `editedAt` property was set on the message but the value was `undefined`. So that was resulting in `"editedAt" in message` to be true :( So to solve this I added this extra check

Reply:
So what about `message?.editedAt && message?.editedBy?._id`? That way you check if keys are there and if they have values 👀

Reply:
That will throw an error since the return of this function is expecting a return type of boolean however `message.editedAt` is of type Date

Reply:
Oh, yeah, didn't mean to copy literally 🙈 , but to remove the "editedAt" in message && "editedBy" in message

Reply:
> I was running into a situation where the editedAt property was set on the message but its value 
do you know how or why is this happening? maybe a migration and fixing the object before insert, if not we should consider the whole typing since `undefined` is allowed

Reply:
@ggazzo I found that the app bridges were the root cause of those undefined properties so I've modified them to prevent this.